SpringProject/src/main/java/com/gradproject2019/conferences/service/ConferenceServiceImpl.java

```diff
@@ -77,11 +77,12 @@ private void checkNotInPast(Instant dateTime) {
             if (!dateTime.isAfter(Instant.now())) {
                 throw new InvalidConferenceFieldException();
             }
-        } catch (NullPointerException ignored) {}
+        } catch (NullPointerException ignored) {
```

Comment:
Why would you ignore NullPointerException? :O

Reply:
This is an old part of the code base so I am not sure exactly who has written it, however I think it is because we are throwing a custom error instead.

Reply:
It might still be useful for you to refactor this in this PR or at least find an explanation why it's written that way, up to you. NullPointerException is thrown when you're trying to use an object that doesn't exist and it's a bad idea to just ignore it.